sensor/tests/context.py

```diff
@@ -14,8 +14,14 @@ def __init__(self):
         self.external_key = str(self.key.external_key)
         self.loc = Location.objects.create( name = "Ulriken" , latitude = 200 , longitude = 120 , altitude = 600)
         self.dev_type = DeviceType.objects.create( name = "HP-X123" )
-        self.user = User.objects.create_user( "test-user" )
         self.channel = "stable"
+        self.user_passwd = get_random_string( length = 10 ),
+        self.user = User.objects.create_user( get_random_string( length = 10 ),
+                                              password = self.user_passwd ,
+                                              first_name = "First",
+                                              last_name = "Last",
+                                              email = "joe@invalid.email.com" )
```

Comment:
could we here have `joe@example.org` instead?  `example.TLD` is meant for such cases.